R/interface.R

```diff
@@ -242,6 +256,20 @@ dust_workdir <- function(path) {
 }
 
 
+dust_rewrite_real <- function(filename, real_t) {
```

Comment:
Should we check that this is either double or float?

Reply:
I wondered but also figured the compiler would whinge. This also allows us later on to fiddle with shorts etc.